### ISSUE TITLE
Is this repo dead?

### ISSUE THREAD

**Issue by attenzione** [ANXS/postgresql#498]
No update for year.
And PostgreSQL 13 have been released.

**Comment by otakup0pe** [comment]
Yeah just about. I've tried to get people to volunteer to help out but there has yet to be enough momentum. Unless some people come out of the woodwork to help I'll likely be archiving several ANXS roles and focus on modernizing others.

**Comment by RCM7** [comment]
Hey @otakup0pe , I use this role in production for a year now and would hate to see it archived. Would love to give a hand maintaining :)

**Comment by otakup0pe** [comment]
that is super rad let's figure out how to chat first and then i can set you up as a maintainer

**Comment by RCM7** [comment]
Feel free to drop me a message on Twitter. Handle in my Github profile ;)

**Comment by maglub** [comment]
I have offered to help maintaining this role through my company (we have 4 good Ansible admins) earlier, but we have gotten the reply "create pull requests".

Our offer is still on. I would happily spend some time and money on at least keeping this repo alive.

**Comment by otakup0pe** [comment]
Noted! I will be reaching out to y'all this week - maglub how should I get ahold of you?

**Comment by maglub** [comment]
> Noted! I will be reaching out to y'all this week - maglub how should I get ahold of you?

I sent you an invite on Linkedin, email address there.

**Comment by egmont1227** [comment]
We (https://github.com/noris-network/) also use this in production - and worked on repmgr extension (https://github.com/Demonware/postgresql/pull/3). Would like to see merges upstream here an willing to help out merging & testing stuff.

**Comment by otakup0pe** [comment]
@egmont1227 awesome, are you on gitter yet?

**Comment by RayOei** [comment]
Slightly related: it would be great if the last v1.12 would be available in the ansible galaxy...

**Comment by egmont1227** [comment]
> @egmont1227 awesome, are you on gitter yet?

Nope. I rather prefer github with async communication ;)

**Comment by otakup0pe** [comment]
I am totally :ok: with github / async comms. Gitter is like that too but now that I think of it I wonder if we could actually use github discussions in it's place.... Something to think about.

I have a soft preference for cleanup around the os support. I know there are a bunch of pr/issues open covering more than just centos8.

**Comment by barazbay** [comment]
I am preparing j2 templates for postgres13, I can push them back to community, if there is none any yet

**Comment by thbar** [comment]
I'll be able to do bit of Postgres 13 testing later (simple install), because I have Ubuntu 16 -> 20 upgrades planned, and will likely group that, if this can help.

**Comment by gclough** [comment]
> No update for year.
> And PostgreSQL 13 have been released.

#504 pushed

**Comment by gclough** [comment]
> I'll be able to do bit of Postgres 13 testing later (simple install), because I have Ubuntu 16 -> 20 upgrades planned, and will likely group that, if this can help.

Ubuntu was mostly broken in this build... and 16.04 is almost EOL anyway, so I pulled it to get builds working again.  If someone (hint, hint, @thbar :) could add it back in for 18 and 20, that would be super.

**Comment by gclough** [comment]
> I am preparing j2 templates for postgres13, I can push them back to community, if there is none any yet

@barazbay , I think a lot of us had done that offline... so I pushed mine in #504 .  Feel free to review and pick it apart if I've made any mistakes.  I did a little tidying up whilst I was there too.

**Comment by thbar** [comment]
@gclough I will attempt to add it back in a couple of weeks, if nobody already took the plunge.

**Comment by maglub** [comment]
@gclough are you in the Gitter channel?

**Comment by otakup0pe** [comment]
> Gitter

I can also spin up a GitHub team under the `ANXS` org so we can use the discussion group functionality, if people would like that. Over the years I've never really seen Gitter get traction for this particular group. I'm willing to go with whatever direction the community is into ✨

**Comment by maglub** [comment]
I think that would be very helpful (discussion functionality). That way we can also start separating help/support from feature requests and bug tickets.

**Comment by maglub** [comment]
Did the Gitter/ANXS die?

![image](https://user-images.githubusercontent.com/2961018/111169587-66746680-85a3-11eb-9a2f-d7a18d18f7f4.png)

**Comment by maglub** [comment]
@otakup0pe, I think setting up a github team would be a good idea. Could you please set it up?

**Comment by otakup0pe** [comment]
I've setup a team on the `ANXS` org for postgresql needs. I also enabled discussions on this repo directly.

**Comment by walbertus** [comment]
Hi, we're also using this role in production. Where can we found the pending task so we can help with it?

**Comment by RayOei** [comment]
Is there any news on this? There seems to be 'some' activity but no new versions thus far?

**Comment by otakup0pe** [comment]
Bumping this thread one last time......

Gitter never really got a ton of traction within this group so I activated the discussion board functionality on the ANXS project. 

And then I made a [post](https://github.com/orgs/ANXS/discussions/1). If anyone is interested in _actively_ helping out with this / other roles - please comment on that discussion. Assuming I set it up properly, everyone should be able to comment. if I didn't, I guess just make a new issue in the repo you wan't to help with and I'll add you to the team.
